src/wac.hpp

```diff
@@ -36,7 +38,7 @@ static WACEntries parse_wac(std::istream &wac_data) {
     WACEntries wac_entries;
     wac_entries.reserve(entry_count);
 
-    for (auto i = 0; i < entry_count; ++i) {
+    for (unsigned i = 0; i < entry_count; ++i) {
```

Comment:
why?

Reply:
auto makes it signed

Reply:
i know lol but I still meant that it's pointless in this case and `auto` is faster to read. but you can just leave it doesn't matter

Reply:
one less warning = good for my eyes